### PR TITLE
motivation and solution for argparse exercise; closes #103

### LINKED ISSUE #103: argparse exercise: motivation not clear and solution missing
we received this feedback from a participant

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

File: scripts.rst [hunk 1]
```diff
@@ -231,6 +231,49 @@ arguments, it also automatically generates a ``--help`` option for you:
...
+      import requests
+      import matplotlib.pyplot as plt
+      import argparse
+
+      parser = argparse.ArgumentParser()
+      parser.add_argument("-i", "--input", type=str, help="input data file")
+      parser.add_argument("-o", "--output", type=str, help="output plot file")
+      args = parser.parse_args()
+
+      url = f"https://raw.githubusercontent.com/swcarpentry/python-novice-inflammation/gh-pages/data/{args.input}"
```
Comment (annefou) [inline scripts.rst:250]: I am not sure I would hard-code the url in the code (and not only have `url = args.input`).

Comment (bast) [reply scripts.rst:250]: Good point. I will change this.

Comment (annefou) [reply scripts.rst:250]: Is line 250 needed?

Comment (bast) [reply scripts.rst:250]: Oops where does that come from :-)

**Review by annefou — APPROVED** [review]
Super! Thank you.

**Pull request merged**